lib/ts/recipe/emailverification/types.ts

```diff
@@ -24,7 +24,15 @@ export type PreAPIHookContext = {
     userContext: any;
 };
 
-export type InputType = RecipeConfig<PreAPIHookAction, PreAPIHookContext> & {
+export type PostAPIHookContext = {
+    action: PreAPIHookAction;
```

Comment:
Consider renaming `PreAPIHookAction` type to `PreAndPostAPIHookAction`

Reply:
Done